gui/storage/migrations/0010_auto_20180618_0340.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-06-18 03:40
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def create_root_dataset(apps, schema_editor):
+    RootDataset = apps.get_model('storage', 'rootdataset')
+    rd = RootDataset()
+    rd.quota_warning = 0.8
+    rd.quota_critical = 0.95
+    rd.refquota_warning = 0.8
+    rd.refquota_critical = 0.95
+    rd.save()
```

Comment:
Do we even need a model to keep track of those?

How about using ZFS user properties for the dataset? They are inherited so we get all of that for free.

Reply:
Great idea, I'll do it!